pypika/terms.py

```diff
@@ -353,6 +353,17 @@ def get_sql(self, **kwargs: Any) -> str:
         return "-{term}".format(term=self.term.get_sql(**kwargs))
 
 
+def escaped_param_str(value: str, quote_char: str) -> str:
+    # Did not find how to do that simply with pypika :(
+    return (
+        value.replace("\\", r"\\")
+        .replace(quote_char, quote_char * 2)
```

Comment:
To add something like this now, it should be configurable behind a feature flag. This gives off big breaking energy 🥲

Reply:
OK, why not, how do you see it. a default option set to escape function by default on which you can override and put a no-op one? 

I am here because I saw some SQL injections on our platform, IMHO, it should be applied by default